vendor.conf

```diff
@@ -90,12 +90,12 @@ google.golang.org/grpc                              25c4f928eaa6d96443009bd84238
 gopkg.in/inf.v0                                     d2d2541c53f18d2a059457998ce2876cc8e67cbf # v0.9.1
 gopkg.in/yaml.v2                                    5420a8b6744d3b0345ab293f6fcba19c978f1183 # v2.2.1
 gotest.tools                                        1083505acf35a0bd8a696b26837e1fb3187a7a83 # v2.3.0
-k8s.io/api                                          40a48860b5abbba9aa891b02b32da429b08d96a0 # kubernetes-1.14.0
-k8s.io/apimachinery                                 d7deff9243b165ee192f5551710ea4285dcfd615 # kubernetes-1.14.0
-k8s.io/client-go                                    6ee68ca5fd8355d024d02f9db0b3b667e8357a0f # kubernetes-1.14.0
+k8s.io/api                                          af9c91bd2759f97bddda2451d5175c758938f12b # kubernetes-1.14.3
+k8s.io/apimachinery                                 6a84e37a896db9780c75367af8d2ed2bb944022e # kubernetes-1.14.3
+k8s.io/client-go                                    b8faab9c5193b482cbf69c78578474b7a40e3e6d # kubernetes-1.14.3
 k8s.io/klog                                         71442cd4037d612096940ceb0f3fec3f7fff66e0 # v0.2.0
 k8s.io/kube-openapi                                 5e45bb682580c9be5ffa4d27d367f0eeba125c7b
-k8s.io/kubernetes                                   641856db18352033a0d96dbc99153fa3b27298e5 # v1.14.0
+k8s.io/kubernetes                                   5e53fd6bc17c0dec8434817e69b04a25d8ae0ff0 # v1.14.3
```

Comment:
actually; looks like this version doesn't match with the version that compose-on-kubernetes is developed against / tested with; https://github.com/docker/cli/pull/1937/files#diff-4061fcef378a6d912e14e2ce162a1995R16

I see the last tagged version of compose-on-kubernetes uses kubernetes 1.14.7 (so is one patch release behind on the latest 1.14.x); I can open a PR upstream to update it to 1.14.8, and once that's tagged we could update venturing for both kubernetes and compose-on-kubernetes